src/ooxml/java/org/apache/poi/xslf/usermodel/XMLSlideShow.java

```diff
@@ -302,14 +302,23 @@ public XSLFSlide createSlide() {
      * Create a blank chart on the given slide.
      */
     public XSLFChart createChart(XSLFSlide slide) {
+        XSLFChart chart = createChart();
+        slide.addRelation(null, XSLFRelation.CHART, chart);
+        return chart;
+    }
+    
+    /**
```

Comment:
added a method to create template for chart as we have method for XWPFDocument and XSSFWorkbook.